app-with-tutorials/cap/fiori-elements-incident-management/README.md

```diff
@@ -0,0 +1,18 @@
+## Description 
+
+This is a sample app to manage incidents. This app is based on SAP Fiori elements list report object for OData V4. 
+
+To learn how to build this app, check out [this](https://developers.sap.com/group.fiori-tools-odata-v4-incident.html) tutorial.
+
+## Requirements
+
+This app is intended as a sample app that can be previwed using SAP Fiori tools in SAP Business Application or VSC. Refer to [documentation](https://help.sap.com/docs/SAP_FIORI_tools/17d50220bcd848aa854c9c182d65b699/2d8b1cb11f6541e5ab16f05461c64201.html) for help on how to get started with SAP Fiori tools. 
```

Comment:
I would write out  Visual Studio Code here since folks may not be familiar with the SBAS/VSC offering if they are new to tools.  Also small typo correction: 
```suggestion
This app is intended as a sample app that can be previewed using SAP Fiori tools in SAP Business Application Studio or Visual Studio Code. Refer to [documentation](https://help.sap.com/docs/SAP_FIORI_tools/17d50220bcd848aa854c9c182d65b699/2d8b1cb11f6541e5ab16f05461c64201.html) for help on how to get started with SAP Fiori tools. 
```